docker/gcp_pudl_etl.sh

```diff
@@ -172,8 +172,10 @@ function update_nightly_branch() {
 function clean_up_outputs_for_distribution() {
     # Compress the SQLite DBs for easier distribution
     gzip --verbose "$PUDL_OUTPUT"/*.sqlite && \
-    # Remove redundant multi-file EPA CEMS outputs prior to distribution
-    rm -rf "$PUDL_OUTPUT/core_epacems__hourly_emissions/" && \
+    # Grab the consolidated EPA CEMS outputs for distribution
+    cp "$PUDL_OUTPUT/parquet/core_epacems__hourly_emissions.parquet" "$PUDL_OUTPUT" && \
+    # Remove all other parquet output, which we are not yet distributing.
+    rm -rf "$PUDL_OUTPUT/parquet" && \
```

Comment:
@zschira I updated this to work with the new output locations. Does this look correct to your?

Reply:
That looks good to me! Anything else before this is ready to merge?

I did run the ETL locally, and had a weird issue that I think is unrelated. I'm investigating/re-running now.

Reply:
I think everything in here looks good to me, but let's wait until we get a good nightly build and can push a release out before we merge in this relatively large change.